Clicker/Views/GroupControls/ExportButtonCell.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ExportButtonCell.swift
+//  Clicker
+//
+//  Created by Mindy Lou on 3/9/19.
+//  Copyright © 2019 CornellAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol ExportButtonCellDelegate: class {
+    func exportButtonCellButtonWasTapped()
+}
+
+class ExportButtonCell: UICollectionViewCell {
+
+    // MARK: View vars
+    var exportButton: UIButton!
+
+    // MARK: Data vars
+    weak var delegate: ExportButtonCellDelegate?
+
+    // MARK: Constants
+    let exportButtonBorderWidth: CGFloat = 1
+    let exportButtonHeight: CGFloat = 47
+    let exportButtonWidthScaleFactor: CGFloat = 0.43
+    let exportButtonTitle = "Export"
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        exportButton = UIButton()
+        exportButton.setTitle(exportButtonTitle, for: .normal)
+        exportButton.titleLabel?.font = ._16MediumFont
+        exportButton.layer.cornerRadius = exportButtonHeight / 2.0
+        exportButton.layer.borderWidth = exportButtonBorderWidth
+        exportButton.addTarget(self, action: #selector(export), for: .touchUpInside)
+        contentView.addSubview(exportButton)
+
+        exportButton.snp.makeConstraints { make in
+            make.width.equalToSuperview().multipliedBy(exportButtonWidthScaleFactor)
+            make.height.equalTo(exportButtonHeight)
+            make.centerX.equalToSuperview()
```

Comment:
same here - combine `centerX.bottom.equalToSuperview()` (this was probably me sorry)